CHANGELOG.yaml

```
@@ -12,6 +12,13 @@ meta:
 
 changes:
 
+  - title: Adding a size_by_type field to the fileFormat aggregate
```

Remember who your audience is. The changelog is for people outside of the Azul team: Mallory, Dave etc.

Your title describes a technical detail on how you solved this (and that description isn't very good either). Instead you need to describe how this affects clients of our service.

```suggestion
  - title: Fix: file type summary is truncated to ten entries
```

---

You can reuse this changelog title as the commit title.